sortinghat/core/schema.py

```diff
@@ -1510,3 +1539,19 @@ class SortingHatMutation(graphene.ObjectType):
     token_auth = graphql_jwt.ObtainJSONWebToken.Field()
     verify_token = graphql_jwt.Verify.Field(description='Verify a JSON Web Token.')
     refresh_token = graphql_jwt.Refresh.Field(description='Refresh a JSON Web Token.')
+
+
+def filter_teams(query, filters):
```

Comment:
Its not working with __ since the resolvers arent able to access it. I have modified the function to be a private function inside the same class as resolvers @sduenas